[DOC] Update Full DeviceConfig Example for release v1.1.0 (#54)
ROCm/gpu-operator: docs/fulldeviceconfig.rst

.. code-block:: yaml
  
    apiVersion: amd.com/v1alpha1 
    kind: DeviceConfig # New Custom Resource Definition used by the GPU Operator
    metadata:

>       # Name of the DeviceConfig CR. Note that the name of device plugin, node-labeller and metric-explorter pods will be prefixed with 

Reviewer: ```suggestion
      # Name that will prefix device plugin, node-labeller and metrics-exporter pods
```

noticed a typo here

      name: gpu-operator

>       # Namespace for the GPU Operator and it's components

Reviewer: ```suggestion
      # Namespace where the GPU Operator and its components will run
```

      namespace: kube-amd-gpu
    spec: 
      ## AMD GPU Driver Configuration ##
      driver:
        # Set enable to false to skip driver installation to use inbox or pre-installed driver on worker nodes
        # Set enable to true to enable operator to install out-of-tree amdgpu kernel module

>         # Default value is true

Reviewer: ```suggestion
        # Set to false to use existing in-tree/pre-installed driver
        # Set to true to install out-of-tree amdgpu kernel module
        # Default: true
``

        nodePort: 32500 # Used instead of `port` when using NodePort as the serviceType. The port number must be between 30000-32767
        # (Optional) Specifying metrics exporter image is optional. Default imagename shown here if not specified.
        image: rocm/device-metrics-exporter:latest # Change this to trigger metrics exporter upgrade on CR update
        # (Optional) Specify image registry secret to pull metrics exporter image if needed. 

>         imageRegistrySecret:
>           name: exporter-image-pulll-secert
>         # (Optional) Specify optional configuration in a configmap for metrics exporter
>         config:
>           name: exporter-configmap
>         # (Optional) Specify optional kube rbac proxy configuration for metrics exporter
>         rbacConfig:
>           # disable by default
>           enable: false
>           # set kube-rbac-proxy image
>           image: "quay.io/brancz/kube-rbac-proxy:v0.18.1"
>           # set to true to disable https protecting the proxy endpoint
>           disableHttps: false
>           # set to true to certificate secret to mount in kube-rbac-proxy container for TLS
>           # self signed certificates will be generated by default
>           secret:
>             name: my-kube-rbac-proxy-cert

Reviewer: ```suggestion
        # Private registry credentials (optional)
        imageRegistrySecret:
          name: exporter-image-pull-secret
        # Custom metrics exporter configuration (optional)
        config:
          name: exporter-configmap
        # RBAC Proxy Configuration for secure metrics endpoint access (optional)
        rbacConfig:
          # Enable RBAC authentication proxy (Default: false)
          # When enabled, provides authentication and authorization for metrics endpoint        
          enable: false
          # RBAC proxy container image
          # Default: quay.io/brancz/kube-rbac-proxy:v0.18.1          
          image: "quay.io/brancz/kube-rbac-proxy:v0.18.1"
          # TLS configuration for metrics endpoint
          # Set true to disable HTTPS          
          disableHttps: false
          # TLS certificate configuration
          # Default: Auto-generated self-signed certificates         
          secret:
            name: my-kube-rbac-proxy-cert
```
